ports/unix/modtime.c

```diff
@@ -67,7 +67,7 @@ static inline int msec_sleep_tv(struct timeval *tv) {
 #endif
 
 STATIC mp_obj_t mod_time_time(void) {
-    #if MICROPY_PY_BUILTINS_FLOAT
+    #if ((MICROPY_PY_BUILTINS_FLOAT) && (MICROPY_FLOAT_IMPL == MICROPY_FLOAT_IMPL_DOUBLE))
```

Comment:
I don't think any of these parenthesis are needed, and to keep a consistent style with the rest of the code would be good to remove the unneeded ones (same below).

Reply:
I see these macros are mostly used without parentheses. So they don't make sense here. I remove them ...